schema/meta-schema.json

```diff
@@ -245,7 +245,11 @@
     },
     "versionId": {
       "type": "boolean",
-      "default": false
+      "default": false,
+      "deprecated": {
```

Comment:
@jpmckinney I'm not sure if I should remove the field entirely or just adding the deprecated object

Reply:
Remove the field entirely. We only use `deprecated` in the actual schema, not the metaschema.